ir/src/validation/error.rs

```diff
@@ -2,6 +2,7 @@ use super::{
     AccessType, ConstrainedBoundary, ConstraintDomain, Symbol, SymbolBinding, TraceAccess,
     TraceSegment, MIN_CYCLE_LENGTH,
 };
+use air_script_core::Expression;
```

Comment:
nit: we should probably follow the same pattern we use elsewhere of importing from super